### PR TITLE
More API Work

### PR DESCRIPTION
**Pull request by MrMagaw**

### CODE REVIEW

File: app/Http/Controllers/BillController.php [hunk 1]
```diff
@@ -126,15 +126,19 @@ public function edit($id) {
     }
 
     public static function getBillsInt($input) {
-        $startDate = isset($input['start_date']) ?
+        $startDate = strtotime(isset($input['start_date']) ?
                 $input['start_date'] :
-                date('Y-m-d G:i:s', strtotime('-' . env('DEFAULT_BILL_AGE', '6 month')));
+                date('Y-m-d G:i:s', strtotime('-' . env('DEFAULT_BILL_AGE', '6 month'))));
```
Comment (Cyrelc) [inline app/Http/Controllers/BillController.php:131]: Appears to be one too many closing braces

File: database/migrations/2016_04_28_092059_create_permissions_roles_pivot.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 class CreatePermissionsRolesPivot extends Migration {
     public function up() {
-        Schema::create('permission_roles', function(Blueprint $table) {
+        Schema::create('permission_role', function(Blueprint $table) {
```
Comment (Cyrelc) [inline database/migrations/2016_04_28_092059_create_permissions_roles_pivot.php:8]: Your plurals have dropped here and reverted to singles again

File: app/Http/Controllers/BillController.php (hunk 1 above)
Comment (MrMagaw) [inline app/Http/Controllers/BillController.php:131]: The last one ends the strtotime on line 129

File: database/migrations/2016_04_28_092059_create_permissions_roles_pivot.php (hunk 2 above)
Comment (MrMagaw) [inline database/migrations/2016_04_28_092059_create_permissions_roles_pivot.php:8]: That's the default naming convention for pivot tables in laravel

**Pull request merged**